# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: c009bbdbbfd5
# Full Hash: c009bbdbbfd5451756cb1f3dc421b82df50911d1
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 22:07:34
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Support preloading also for index cursors. r=ttung,asuth
#   
#   Depends on D43252
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43461
# ==============================================================================

diff -r 89817a19f3b3 -r c009bbdbbfd5 dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:37 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 14:40:35 2019 +0000
@@ -80,7 +80,7 @@
   nsresult;
   ObjectStoreCursorResponse[];
   ObjectStoreKeyCursorResponse;
-  IndexCursorResponse;
+  IndexCursorResponse[];
   IndexKeyCursorResponse;
 };
 
@@ -91,7 +91,8 @@
 parent:
   async DeleteMe();
 
-  async Continue(CursorRequestParams params, Key currentKey);
+  async Continue(CursorRequestParams params, Key currentKey,
+                 Key currentObjectStoreKey);
 
 child:
   async __delete__();
